static/js/notes.js

```diff
@@ -71,68 +71,32 @@ const $createTextAreaDiv = function(labelText) {
 
 
 // NOTES
-const $createAuthor = function(authorId) {
-  const user = USERS_BY_ID[authorId];
-
-  return $('<span>', {
-    class: classNamesByName.noteAuthorSpan,
-    text: user.displayName
-  });
+function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
 };
 
-const $createType = function(noteId) {
-  let spandId = noteId + "_category";
+function assignCategory(noteId, $cateogry) {
```

Comment:
Totally unnecessary for this pr,  but when a function does some async action, i'd consider returning the promise from the function or making it an async function
```javascript
function assignCategory(noteId, $cateogry) {
  let id = `${noteId}_category`;
  $cateogry.prop('id', id);

  return $.get('/api/note-category?id=' + noteId).then( (responseObj) => {
      $(`#${id}`).text(capitalizeFirstLetter(responseObj.category));
  });
}
```

or
```javascript
async function assignCategory(noteId, $cateogry) {
  let id = `${noteId}_category`;
  $cateogry.prop('id', id);

  responseObj = await $.get('/api/note-category?id=' + noteId)
  $(`#${id}`).text(capitalizeFirstLetter(responseObj.category));
}
```

that'll be useful later if the caller of `assignCategory` wants to do something after the category is assigned. (Of course you can always change `assignCategory` once the caller wants to do something, but if it were me i'd probably modify `assignCategory` now)